Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml

```
@@ -0,0 +1,89 @@
+args:
+- description: List of users IDs of the user to retrieve.
+  name: user_id
+  isArray: true
+- description: List of users names of the user to retrieve.
```

```suggestion
- description: List of names of the users to retrieve.
```
Should this be:  A comma-separated list of names of the users to retrieve?